plugins/inputs/execd/execd_test.go

```diff
@@ -355,7 +355,7 @@ func readChanWithTimeout(t *testing.T, metrics chan telegraf.Metric, timeout tim
 	case m := <-metrics:
 		return m
 	case <-to.C:
-		require.FailNow(t, "timeout waiting for metric")
+		require.FailNow(t, "Timeout waiting for metric")
```

Comment:
Hmmm `FailNow` is a tricky beast as it will prevent `defer`red functions to run. Is this really what we want here?